src/lib/command_line_tests/command_line_tests.ml

```diff
@@ -0,0 +1,62 @@
+open Core
+open Async
+
+(**
+ * Test the basic functionality of the coda daemon and client through the CLI
+ *)
+
+let%test_module "Command line tests" =
+  ( module struct
+    (* executable location relative to src/default/lib/command_line_tests
+
+       dune won't allow running it via "dune exec", because it's outside its
+       workspace, so we invoke the executable directly
+
+       the coda.exe executable must have been built before running the test
```

Comment:
Can we add an explicit test for the existence of `coda.exe`, with an error message to run `dune build`? I just want to be friendlier to wrong uses of `dune runtest`.